keycloak/keycloak-operator
# Allow Realm Password policy to be defined in the keycloak realm CRD - KEYCLOAK-17607

##  https://issues.redhat.com/browse/KEYCLOAK-17607

## Additional Information
A customer wants the passwrodPolicy  to be implemented.  have forked an operator for them and created this PR.

## Verification Steps
Run the operator in local mode make code/run
Create a keycloakrealm object as per deploy/examples/realm/password_policy_realm.yaml and verify that the password policy has been set under the Authentication tab in the left menu GUI and select the Password Policy tab on the right side. Verify that changes are there
